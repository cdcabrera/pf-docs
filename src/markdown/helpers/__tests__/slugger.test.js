const { slugger, makeSlug } = require('../slugger');

describe('Slugger', () => {
  it('should makeSlug', () => {
    expect(makeSlug('Lorem source', 'Ipsum section', 'Dolor ID')).toMatchSnapshot('makeSlug');
  });

  it('should slugger', () => {
    expect(slugger(['Lorem ipsum', 'Dolor sit'])).toMatchSnapshot('slugger');
  });
});
