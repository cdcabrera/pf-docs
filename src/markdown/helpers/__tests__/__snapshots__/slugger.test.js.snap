// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slugger should makeSlug: makeSlug 1`] = `"/ipsum-section/dolor-id/Lorem source"`;

exports[`Slugger should slugger: slugger 1`] = `"lorem-ipsumdolor-sit"`;
